addons/ui/ACE_UI.hpp

```diff
@@ -0,0 +1,177 @@
+// IDD, elements[]s, Element (must be string to compare to changed setting name), Selective Type (0-both, 1-ground, 2-vehicle)
+class ACE_UI {
+    class weaponName {
+        idd = 300;
+        elements[] = {118};
+        condition = QUOTE(ACE_player == vehicle ACE_player);
+        class conditions {};
```

Comment:
https://github.com/acemod/ACE3/pull/3805/files#diff-fc2d593f082cd4c729bb26d91f30ede9R31

The only thing is that no other condition gets evaluated if that one is false. I could put it into conditions class and filter it out in `configProperties` as well if that would make it nicer.